hotel_res.sql

```diff
@@ -59,10 +109,46 @@ CREATE TABLE employee
 	 foreign key (location_address) references location,
 	 foreign key (manager_id) references employee);
 
+	CREATE SEQUENCE e_seq;
```

Comment:
@jasdeepk not an error or anything, but I'm just curious what this "CREATE SEQUENCE e_seq" and all the code after it does?